docs/code/os-dev.md

```diff
@@ -0,0 +1,56 @@
+# elementary OS Development
+Want to contribute code to elementary OS itself? Here are some tips.
+
+## Vala linting
+To make it easier to follow the [elementary Code-Style guidelines](https://elementary.io/docs/code/reference#code-style) you can use [Vala-lint](https://github.com/elementary/vala-lint).
+
+## GtkInspector
+Use the [GtkInspector](https://wiki.gnome.org/Projects/GTK+/Inspector) to make development easier.
```

Comment:
This should probably cover how to get it (install 'libgtk-3-dev`, which is included in `elementary-sdk` and then set `org.gtk.Settings.Debug enable-inspector-keybinding` to `true`) instead of just linking to a GNOME Wiki.

Reply:
I didn't want to duplicate their documentation, but sure, I'll add it.

Reply:
@peteruithoven their documentation can change and doesn't necessarily imply elementary OS, whereas we know exactly the steps required for our platform (like installing the SDK).

Reply:
Good point